lib/features/profile/presentation/widgets/edit_profile_image_widget.dart

```diff
@@ -0,0 +1,32 @@
+import 'package:final_project/core/presentation/theme/colors.dart';
+import 'package:final_project/features/profile/presentation/widgets/profile_image_widget.dart';
+import 'package:flutter/material.dart';
+
+class EditProfileImageWidget extends StatelessWidget {
```

Comment:
Don't use suffix like `Widget`. If it extends `Widget` it is a `Widget`. Try a more meaningfull name.